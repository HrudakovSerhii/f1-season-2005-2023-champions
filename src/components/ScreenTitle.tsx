import React from 'react'

const ScreenTitle: React.FC<{ title: string }> = ({ title }) => {
    return (
        <div role="textbox" className="w-full max-w-screen-xl p-4">
            <h2 className="text-lg text-gray-600 font-medium">{title}</h2>
        </div>
    )
}

export default ScreenTitle
