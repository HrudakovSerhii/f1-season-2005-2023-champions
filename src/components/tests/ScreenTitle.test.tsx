import React from 'react'
import { render } from '@testing-library/react'

import ScreenTitle from '../ScreenTitle'

describe('App ScreenTitle component', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<ScreenTitle title="Clean title" />)

        expect(asFragment()).toMatchSnapshot()
    })
})
