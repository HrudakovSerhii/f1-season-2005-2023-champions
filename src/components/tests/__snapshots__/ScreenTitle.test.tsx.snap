// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App ScreenTitle component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="w-full max-w-screen-xl p-4"
    role="textbox"
  >
    <h2
      class="text-lg text-gray-600 font-medium"
    >
      Clean title
    </h2>
  </div>
</DocumentFragment>
`;
